test/regression-wasm/test-simple-arithmetic-spec.kore

```diff
@@ -1,39 +1,39 @@
-[org'Stop'kframework'Stop'attributes'Stop'Source{}("Source(wasm-semantics/tests/proofs/simple-arithmetic-spec.k)")]
+[org'Stop'kframework'Stop'attributes'Stop'Source{}("Source(/Users/emarzion/kore/wasm-semantics/tests/proofs/simple-arithmetic-spec.k)")]
```

Comment:
Also a bunch of new references to `/Users/emarzion/kore` in this file.

Actually I think it might be all the `.kore` files. They should probably all be looked at. (They won't all load in the github browser because they are large).